### PR TITLE
add contribution guidelines

### PR DESCRIPTION
**Pull request by thomas-fossati**
Ironically, this PR contradicts itself by not having a linked issue :-)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Veraison components - contribution guidelines
+
+Contributions to this project are welcomed. We request that you read through
+the guidelines before getting started.
+
+## Community guidelines
+
+Get acquainted with [CONTRIBUTING.md](CONTRIBUTING.md) that contains our
```
Comment (SimonFrost-Arm) [inline CONTRIBUTING.md]: Self recurse?

Comment (SimonFrost-Arm) [inline CONTRIBUTING.md]: Self recurse?

Comment (SimonFrost-Arm) [reply CONTRIBUTING.md]: Ah, fixed in part 2

**Review by SimonFrost-Arm — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Veraison components - contribution guidelines
+
+Contributions to this project are welcomed. We request that you read through
```
Comment (mangup01) [inline CONTRIBUTING.md:3]: Extra 'd' in "welcome"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Run the tests with:
+
+```text
+$ go test ./...
+```
+
+### Presubmit checks
+
+TODO(tho) we probably should have a `scripts/presubmit.sh` which mirrors the
```
Comment (mangup01) [inline CONTRIBUTING.md:50]: Shall we replace "probably should have" to "In future we shall have"

File: CONTRIBUTING.md (hunk 2 above)
Comment (thomas-fossati) [reply CONTRIBUTING.md:3]: ack

File: CONTRIBUTING.md (hunk 3 above)
Comment (thomas-fossati) [reply CONTRIBUTING.md:50]: ack

**Review by mangup01 — APPROVED** [review]

**Pull request merged**